autopilot/agent.go

```diff
@@ -578,18 +579,58 @@ func (a *Agent) openChans(availableFunds btcutil.Amount, numChans uint32,
 	)
 	a.pendingMtx.Unlock()
 
-	// If we reach this point, then according to our heuristic we
-	// should modify our channel state to tend towards what it
-	// determines to the optimal state. So we'll call Select to get
-	// a fresh batch of attachment directives, passing in the
-	// amount of funds available for us to use.
-	chanCandidates, err := a.cfg.Heuristic.Select(
-		a.cfg.Self, a.cfg.Graph, availableFunds,
-		numChans, nodesToSkip,
+	// Gather the set of all nodes in the graph, except those we
```

Comment:
Typo in commit message: copmatible -> compatible

Reply:
Fixed.